src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationHandler.cs

```diff
@@ -76,6 +76,9 @@ public class OdnoklassnikiAuthenticationHandler : OAuthHandler<OdnoklassnikiAuth
             return new AuthenticationTicket(context.Principal!, context.Properties, Scheme.Name);
         }
 
+        protected override string FormatScope([NotNull] IEnumerable<string> scopes)
+            => string.Join(";", scopes);
```

Comment:
In .NET 5 we can leverage the overload that takes a `char`.

```suggestion
            => string.Join(';', scopes);
```